_rules/visible-label-in-accessible-name-2ee8b8.md

```diff
@@ -18,77 +18,133 @@ acknowledgements:
 
 ## Accessibility Support
 
-There are no major accessibility support issues known for this rule.
+_No accessibility support issues known._
 
 ## Background
 
 - [Understanding Success Criterion 2.5.3: Label in Name](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html)
+- [Technique G208: Including the text of the visible label as part of the accessible name](https://www.w3.org/WAI/WCAG21/Techniques/general/G208)
+- [Technique G211: Matching the accessible name to the visible label](https://www.w3.org/WAI/WCAG21/Techniques/general/G211)
 
 ## Test Cases
 
 ### Passed
 
 #### Passed Example 1
 
-[Visible][] label and [accessible name][] matches when trailing white spaces are removed.
+[Visible][] [label][] and [accessible name][] match when trailing white spaces are removed.
 
 ```html
 <div role="link" aria-label="next page ">next page</div>
 ```
 
 #### Passed Example 2
 
-Character insensitivity between [visible][] label and [accessible name][].
+[Visible][] [label][] and [accessible name][] match when ignoring case sensitivity.
 
 ```html
 <div role="link" aria-label="Next Page">next page</div>
 ```
 
 #### Passed Example 3
 
-Full [visible][] label is contained in the [accessible name][].
+Full [visible][] [label][] is contained in the [accessible name][].
+
+```html
+<button name="link" aria-label="View next Page in the list">Next Page</button>
+```
+
+#### Passed Example 4
+
+Full [visible][] [label][] is contained in the [accessible name][], though the name contains more text than the [visible][] [label][].
+
+```html
+<head>
+	<style>
+		.sr-only {
+			position: absolute;
+			left: -10000px;
+			top: auto;
+			width: 1px;
+			height: 1px;
+			overflow: hidden;
+		}
+	</style>
+</head>
+
+<body>
+	<a href="https://github.com/pricing">Read more<span class="sr-only"> about our pricing</span></a>
+</body>
+```
+
+#### Passed Example 5
+
+[Descendant text content](https://dom.spec.whatwg.org/#concept-descendant-text-content) does not express anything in [human language](https://www.w3.org/TR/WCAG21/#dfn-human-language-s) and do therefore not live up to the [WCAG definition of text](https://www.w3.org/TR/WCAG21/#dfn-text).
+
+```html
+<button aria-label="close">:-)</button>
+```
+
+#### Passed Example 6
+
+[Visible][] [label][] and [accessible name][] match when punctuation is removed.
 
 ```html
-<button name="link" aria-label="Next Page in the list">Next Page</button>
+<div role="link" aria-label="Next page">Next page...</div>
+```
+
+#### Passed Example 7
+
+[Visible][] [label][] and [accessible name][] match when parts that do not express anything in human language, "X", and punctuation are removed.
```

Comment:
```suggestion
The [visible][] [descendant text content][] and [accessible name][] match when parts that do not express anything in human language, ("X"), and punctuation are removed.
```
I think it would be better to use a Unicode symbol for cross/closing rather than a letter. That would illustrate more clearly that it is not human language (but some sort of icon).